sdlib/d/gc/emap.d

```diff
@@ -104,8 +107,8 @@ public:
 		return data >> 60;
 	}
 
-	auto next() const {
-		enum Increment = 1UL << 60;
+	auto next(ulong pages = 1) const {
+		ulong Increment = pages << 60;
```

Comment:
page count are always uint int he codebase (and for cause, it is packed as such in various places).

variable are not capitalized, one constants are.